vru_use_case_scripts/plot_detected_objects

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser
+from collections import defaultdict
+import csv
+import json
+from pathlib import Path
+import sys
+
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+from scipy.interpolate import splrep, BSpline
+
+
+def parse_positions_from_sdsm_csv(file_path: Path):
+    def get_timestamp(csv_row: list):
+        return round(float(csv_row[1]))
```

Comment:
Is this a problem with my  parsing script that should get fixed.

Reply:
I did this for correctness. The simulation time values from the `sdsm.csv` file are floating-point, but they look like `123.000045353`, `124.000949531`, _etc._

Reply:
We should probably fix that in a separate issue. Probably won't happen now but at some point in the future.